helm/aws-operator/templates/_helpers.tpl

```diff
@@ -0,0 +1,39 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "aws-operator.name" -}}
+{{- .Chart.Name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "aws-operator.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Common labels
+*/}}
+{{- define "aws-operator.labels" -}}
+helm.sh/chart: {{ include "aws-operator.chart" . }}
+{{ include "aws-operator.selectorLabels" . }}
+app.giantswarm.io/branch: {{ .Values.project.branch }}
+app.giantswarm.io/commit: {{ .Values.project.commit }}
+app.kubernetes.io/name: {{ include "aws-operator.name" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+{{- if .Chart.AppVersion }}
+{{ include "aws-operator.name" . }}.giantswarm.io/version: {{ .Chart.AppVersion }}
```

Comment:
Question: Why does this line not have the `| quote` filter applied? Is it different than L#28 or do we or should we drop `quote` altogether?

Reply:
It probably should to handle some [edge cases](https://stackoverflow.com/a/22235064) (like value of `AppVersion` containing some of the YAML special characters or being just a standalone number). However I'm currently wondering if we should have that label here at all. I asked a question in giantswarm/fmt#100 we'll see, if it turns out we need this here I'll fix it to add `| quote` filter. Thanks for pointing that out :+1: